### DISCUSSION TITLE
Save user inputs in localStorage/IndexedDB

### DISCUSSION THREAD

**Discussion by bogosj** [adamghill/fediview#2] [Ideas]
I'd like there to be a checkbox where I can optionally save all of the inputs (profile and token mainly) in my browser so the next time I come back I can just hit the button again.

I'd be happy to hack at this if you're interested in adding it.

**Comment by adamghill** [comment]
I've been thinking about this as well. Pretty sure localstorage would work great. I'd love a PR if you are interested.

**Reply by bogosj** [reply]
I decided to start with only saving the instance and token values.

https://github.com/adamghill/fediview/pull/10

**Reply by adamghill** [reply]
I merged this in, thank you so much. 🎉

**Comment by bogosj** [comment]
Something appears broken with this now. If I visit the site and hit Get Popular Toots I get this error message. I have to go into the text boxes and make a change (add then remove a space) and then it will work. Note, the token in the screenshot is modified.

![image](https://user-images.githubusercontent.com/10186620/210408227-2067fb18-35cd-40da-a2f5-b552772dc170.png)

**Reply by adamghill** [reply]
Yep, I think I broke this, but I think I know how to fix it. Will get it fixed up asap.

**Reply by adamghill** [reply]
I think this should be fixed in 149a404 which is getting deployed right now.
